JWBestPracticeApps/FeedTableViewController/FeedTableViewController.swift

```diff
@@ -29,8 +29,13 @@ class FeedTableViewController: UITableViewController {
         
         // Populate the feed array with video players
         for itemInfo in feedInfo {
-            if let config = JWConfig.init(contentUrl: itemInfo["url"]),
-                let player = JWPlayerController.init(config: config) {
+            guard let url = itemInfo["url"] else {
```

Comment:
Although the code is correct, i feel like this is not as concise and compact as before. Why can't we use if let..., let..., etc to have all condition check in one if statement?

Reply:
I admit it's not as compact as before. My reasoning behind writing it like this is the 'if let' would be wrapped around the entire body of the loop, and would become further nested by the 'if let player' block. This could look a bit convoluted itself, since the guard can be used to reject the invalid url right off the bat, leaving the rest of the loop's code more readable.

Another option would be to place all of the guard's code on one line. Usually my style is to use multiple lines, but a one-liner guard (e.g. guard let url = itemInfo["url"] { continue }) would look more concise.

What are you thoughts?

Reply:
JWConfig in the previous version JWPlayerSDK 3.6.1 accepts in its `initContentUrl:` an optional url and the initialization was marked as nullable. Now in JWPlayerSDK 3.6.2 the JWConfig non accepts an optional url and also the class is marked as non-null. So that why it doesn't require to be guarded.

Reply:
As I understand it, the Best Practices application is to be built against 3.6.0. In this version, JWConfig does not accept a nullable url, nor does the constructor return a null object. Sadly, because of this, I cannot make this loop more concise using the if let..., let ..., let ... notation.

Reply:
Nevermind. I spoke with Kamil and now this is making sense. I believe my machine is setup incorrectly. I'll speak to Henry on this to be sure. Thanks David!

Reply:
Yeah is something related to the Podfile.lock, you have to make `pod update` instead `pod install` the first time you clone it. I think there is already a ticket to fix that.